Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -250,6 +250,23 @@ void ResourceLoader::load(LoadRequest& request, SuccessCallback success_callback
             return;
         }
 
+        // // When resource uri is a directory use file directory loader to generate response
```

Comment:
You have 2 `//` here

Reply:
Please also capitalize "URI" here, and also capitalize "URL" in the commit message